exampleSite/config.yaml

```diff
@@ -19,6 +19,9 @@ permalinks:
 imaging:
   quality: 99
 
+# To disable, set to a value larger then the maximum number posts/articles on your site (i.e. 100)
+paginate: 5
```

Comment:
Thanks for clarifying the comment. 

I want pagination disabled by default on this theme. Would you set the value to 999999, or something similar to make it obvious

Reply:
Sounds great!  Updated.